bangazonapi/models/product.py

```diff
@@ -61,9 +61,9 @@ def average_rating(self):
         total_rating = 0
         for rating in ratings:
             total_rating += rating.rating
-
-        avg = total_rating / len(ratings)
-        return avg
+        if len(ratings):
```

Comment:
This looks good but the ticket is asking for a try/except block instead

Reply:
Changes have been made